experiments/kan_benchmark/README.md

```diff
@@ -0,0 +1,72 @@
+# KAN for Forecasting - Benchmark on M3 and M4 datasets
+
+[Kolmogorov-Arnold Networks](https://arxiv.org/abs/2404.19756) (KANs) is an alternative to the multilayer perceptron (MLP). In this experiment, we assess the performance of KANs in forecasting time series.
+
+We use the M3 and M4 datasets, which represents more than 102 000 unique time series covering yearly, quarterly, monthly, weekly, daily and hourly frequencies.
+
+While KANs reduce the number of parameters by 38% to 92% compared to the MLP, it also rarely performs better than the MLP in time series forecasting tasks. In this benchmark, N-BEATS still performs best across the vast majority of datasets.
```

Comment:
Cool experiment! We could think about doing the experiment on a similar FLOP / parameter budget as MLP (this is just my science head thinking, no need to do it now, maybe to keep in mind for future benchmarks / work)